docs/admin/terraform/README.md

```diff
@@ -76,11 +76,9 @@ Let's have an example on `importing` the current `logging level` of a deployment
     - **VM:** grab `client_id` and `client_pw` from `/opt/jans/jans-setup/setup.properties.last` file.
 
     - **K8s:** 
-        1. client_id: `kubectl get cm cn -n <namespace> -o yaml | grep test_client_id` 
+        1. client_id: `kubectl get cm cn -n <namespace>  --template={{.data.jca_client_id}}` 
```

Comment:
you can now use the `test_client_id` as thats working. You can add the jca as a fallback